evaluations/run_evaluation.py

```diff
@@ -28,7 +28,7 @@
 
 from wfa_cardinality_estimation_evaluation_framework.evaluations import analyzer
 from wfa_cardinality_estimation_evaluation_framework.evaluations import evaluator
-from wfa_cardinality_estimation_evaluation_framework.evaluations.data import evaluation_configs
+from data import evaluation_configs
```

Comment:
Can you explain why this is changing?

Reply:
Correct me if I'm wrong because I don't have full understanding of the behavior of the repo yet, but it seems that evaluation_configs is pretty tightly coupled with run_evaluation.py. If the 'data' package were inside of 'src/evaluations', then changing the config would require a reinstallation of the package to allow the change to be accessed by run_evaluation.py.

Reply:
Generally we use fully qualified imports.